packages/@aws-cdk/assert/test/test.assertions.ts

```diff
@@ -205,11 +206,19 @@ function failingExample(title: string, cb: (test: Test) => void) {
   };
 }
 
-function synthesizedStack(fn: (stack: cdk.Stack) => void): cx.SynthesizedStack {
+function synthesizedStack(fn: (stack: cdk.Stack) => void): cx.ICloudFormationStackArtifact {
   const app = new cdk.App();
   const stack = new cdk.Stack(app, 'TestStack');
   fn(stack);
-  return app.synthesizeStack(stack.name);
+
+  const session = app.run();
+
+  const assembly = new CloudAssembly(session.outdir);
+  const s = assembly.stacks.find(x => x.name === stack.name);
```

Comment:
Would name `stack`.

Reply:
renamed to `artifact` (we already have a `stack` in this scope).